pkg/configuration/configuration.go

```diff
@@ -124,11 +131,62 @@ func loadFromSecret(cl client.Client) error {
 	return nil
 }
 
+// loadFromConfigMap retrieves the host operator configMap
+func loadFromConfigMap(cl client.Client) error {
+	// get the configMap name
+	configMapName := getResourceName("HOST_OPERATOR_CONFIG_NAME")
```

Comment:
Should be `HOST_OPERATOR_CONFIG_MAP_NAME` , right?
```suggestion
	configMapName := getResourceName("HOST_OPERATOR_CONFIG_MAP_NAME")
```

Reply:
good catch! thanks @alexeykazakov fixed here: eab40417d9d02d007255ae0c75da627258b2a03b